### ISSUE TITLE
ZIP downloads of all papers

### ISSUE THREAD

**Issue by tatecarson** [NIME-conference/nime-website#17]
This link seems to be broken: https://www.nime.org/proceedings/ZIPs/. Also I am looking for the links on the site to the papers from the AWS bucket but they all seem to be linked to zenodo. I am trying to find them so I can try to build the new paper search function. What am i missing?

**Comment by cpmpercussion** [comment]
that link was a bit of a holdover from our previous website, I decided not to add the ZIPs given that I have to pay for bandwidth on those downloads.

Here's a dropbox link with the same result: https://www.dropbox.com/sh/durkmf9px09ieo0/AADqyAJyP4rvfv-ZfML2NoDfa?dl=0

Actually having a decentralised way to download the full zipped proceedings is not a bad idea.

**Comment by alexarje** [comment]
Yes, linking to a dropbox folder is a good way of solving this. I think downloading zip-files is probably most relevant for those that want to do text mining stuff on the entire collection. So I think we should make the link available for people. Are you ok with sharing it from your dropbox?

**Comment by stefanofasciani** [comment]
Zip are available from 2 sources
https://zenodo.org/communities/nime_conference_archive/records?q=&l=list&p=1&s=10&sort=newest
https://www.nime.org/proceedings/ZIPs/
